lib/cob_web_index/indexer_config.rb

```diff
@@ -77,15 +77,30 @@
     acc << rec.fetch("type")
   end
 
-  if rec.fetch("type") == "building"
-    acc << "Library"
-  end
-
   if rec.fetch("type") == "finding_aid"
     acc << "Finding Aids"
   end
 }
 
+to_field "web_content_type_boost_i", ->(rec, acc) {
+  if rec.fetch("type") == "event" || rec.fetch("type") == "highlight"
+    acc << 1
+
+  elsif rec.fetch("type") == "building"
+     acc << 3
+
```

Comment:
@sensei100 should this be another number?  Otherwise it's redundant with "else" below which would assign three to it.

Reply:
You are right, I took it out.